src/main/kotlin/app/revanced/patches/music/player/replace/ReplaceCastButtonPatch.kt

```diff
@@ -32,7 +32,7 @@ import com.android.tools.smali.dexlib2.iface.reference.MethodReference
 
 @Patch(
     name = "Replace cast button",
-    description = "Replace the cast button in the player with the open music button.",
+    description = "Adds an option to replace the cast button in the player with the \"Open music\" button.",
```

Comment:
I don't know what this patch looks like when it works as intended, but I assume that "Open music" should be put in double quotes like mentions of specific elements in other patch descriptions.